docker-compose.test.yml

```diff
@@ -28,7 +28,19 @@ services:
       dockerfile: Dockerfile.test
     env_file:
       - ./conf/test.env
+    environment:
```

Comment:
Is it possible to move these vars to new env file `./conf/travis.env`? I think it's better to keep this file as brief as possible.

Reply:
Actually, why do we need all these vars when they are not set to any value?

Reply:
That means these variables and their values are passed from environment. Meaning when running on travis, these will be set. I don't think variables without values can be specified in `./conf/travis.env`.